src/libumf_linux.c

```diff
@@ -1,6 +1,6 @@
 /*
  *
- * Copyright (C) 2024 Intel Corporation
+ * Copyright (C) 2023-2024 Intel Corporation
```

Comment:
Is it necessary? In `libumf_windows.c` there is just `2024`.
If it's desired please resolve my comment.

Reply:
Done